spec/spec_helper.rb

```diff
@@ -5,13 +5,13 @@
 ENV["RAILS_ENV"] ||= 'test'
 require File.expand_path("../../config/environment", __FILE__)
 require 'rspec/rails'
-require 'rspec/its'
 require 'capybara/rails'
 require 'capybara/rspec'
 require 'capybara/poltergeist'
-require 'factory_girl_rails'
-require File.join(Sufia::Engine.root, 'spec/factories/generic_files')
-require File.join(Sufia::Engine.root, 'spec/support/features')
```

Comment:
This pulls in a bunch of helpers from sufia which we are only using the cleaner for cleanup_jetty. Which is a custom implementation. Let's remove this and create our own cleanup_jetty (I am using ActiveFedora::Cleaner.clean! to do this instead).